3-concepts/score-normalization.md

```diff
@@ -32,31 +30,31 @@ of **0.80** can be interpreted as a belief that **80%** of people would consider
 the comment to be toxic.
 
 However, as we introduce additional data, the basic distribution of how many
-toxic comments there are can change. Because models are far from perfect, this
+toxic comments there are can change. Because attributes are far from perfect, this
 would result in a shift in the calibration scores. To control for this, we
 perform callibration on a randomly selected 50/50 class split. of course, this
 control is not perfect: if the newly introduced examples are differently
 difficult to previous ones, some change in scores is still likely.
 However, in practice, we think this will be reasonably stable, and clients will
 be able to have the same 'sense' of what a threshold means.
 
-
 We are always looking for better ways to provide stability, so if you know of
 better ways to do this, please
 [open an issue on GitHub](https://github.com/conversationai/perspectiveapi/issues).
 
 ## About score feedback
 
-Our models are not perfect and will make errors. It will be unable to detect patterns
-of toxicity it has not seen before, and it will falsely detect comments similar to
-patterns of previous toxic conversations. To help improve the machine learning, the API
-supports sending our team suggested scores and corrections.
+Our attributes are not perfect and will make errors. It will be unable to
```

Comment:
In this case (and in the paragraph above) it might make more sense to keep model, or say "model attributes" or "machine learning attributes" since when we talk about errors, it's usually in the context of models. Attributes are the *names* we give our model results in the API responses, but it's not the attributes making the errors, it's the models.